### DISCUSSION TITLE
Windows 10 search box - request

### DISCUSSION THREAD

**Discussion by ghost** [valinet/ExplorerPatcher#641] [General]
Hey I love the Explorer patcher, it looks exactly like the Windows 10 Taskbar we are all familiar too. One question would there be any chance you could add the rectangler search bar back that would be awesome.

**Comment by valinet** [comment]
Not really. Best replacement would be [Everything Toolbar](https://github.com/stnkl/EverythingToolbar). The original bar is anyway useles other than cosmetics, it acts mostly the same as the search button.
